Cargo.toml

```diff
@@ -17,7 +17,8 @@ rust-version = "1.74"
 [features]
 default = []
 client = ["dep:reqwest", "dep:url"]
-server = ["dep:actix-cors", "dep:actix-web"]
+server = ["dep:time", "dep:actix-cors", "dep:actix-web"]
```

Comment:
I'm not sure why `dep:time` is being added since you're not modifying the `server` component?

Reply:
See below